src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -218,6 +218,7 @@ function MdAutocomplete ($$mdSvgRegistry) {
         // Retrieve the state of using a md-not-found template by using our attribute, which will
         // be added to the element in the template function.
         ctrl.hasNotFound = !!element.attr('md-has-not-found');
+        $mdTheming(element);
```

Comment:
Why do we need to do this again? The controller already does register the theme.

> https://github.com/angular/material/blob/master/src/components/autocomplete/js/autocompleteController.js#L70